dbt/models/model/docs.md

```diff
@@ -138,7 +138,7 @@ The test set is the out-of-sample data used to evaluate model performance.
 Predictions in this table are trained using only data _not in this set
 of sales_.
 
-**Primary Key**: `year`, `run_id`, `meta_pin`, `meta_card_num`
+**Primary Key**: `year`, `run_id`, `meta_pin`, `meta_card_num`, `document_number`
```

Comment:
**nitpick (non-blocking):** There's not (yet) a doc number column in this dataset.

Reply:
This is a variable do we want it?
meta_sale_document_num